integration-tests/utils/integration-utils.js

```diff
@@ -5,6 +5,8 @@ const { Storage } = require('@google-cloud/storage')
 const { screenshotBucket, screenshotDirPath } = require('../utils/integration-config')
 
 
+const waitForXpathWithVisibleTrue = opts => _.defaults({ visible: true }, opts)
```

Comment:
Can we rename the function to something that more closely matches what it does, e.g. `defaultVisibleToTrue`?

Note also that explicitly specifying input (`opts`) is unnecessary.

```suggestion
const defaultVisibleToTrue = _.defaults({ visible: true })
```

Reply:
Thanks, updated code.